### ISSUE TITLE
[Feature/Idea]: add hint that you need to enter `y` (yes) or `n` (no)

### ISSUE THREAD

**Issue by rakleed** [russellbanks/Komac#482]
### What would you like to see changed/added?

Add a hint to enter `y` (yes) or `n` (no) in questions where only such answers are accepted. Now, in order to see it, you need to not fill out the field once or enter an incorrect value, which is inconvenient.

For example, `> Is <package> a portable exe?` or `? Would you like to make a pull request for <package>?`

**Comment by CoolPlayLin** [comment]
Yes, and you also can add an argument which is called `-y`. It is used to agree all such similar question by default. Personally, I think it can help me submit package more quickly
